lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -318,5 +318,5 @@ def _callback_render(self, result, request, id_, version, auth_required=False):
             log.exception("Failed to render API response: %s", result)
             self._render_error(err, request, id_, version)
 
-    def _render_response(self, result, code):
-        return defer.succeed({'result': result, 'code': code})
+    def _render_response(self, result):
+        return defer.succeed({'result': result, 'code': 200})
```

Comment:
even here, the code is never used.  In _callback_render, it only pulls out the 'result'